docs/installation/install_epinio.md

```diff
@@ -58,67 +69,55 @@ certificates for the various Ingresses (see "Ingress controller" above).
 
 If cert-manager is not already installed on the cluster, it can be installed like this:
 
+```bash
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm upgrade --install cert-manager jetstack/cert-manager --namespace cert-manager  \
+    --set installCRDs=true \
+    --set extraArgs={--enable-certificate-owner-ref=true} \
+    --create-namespace
 ```
-$ kubectl create namespace cert-manager
-$ helm repo add jetstack https://charts.jetstack.io
-$ helm repo update
-$ helm install cert-manager --namespace cert-manager jetstack/cert-manager \
-		--set installCRDs=true \
-		--set extraArgs={--enable-certificate-owner-ref=true}
+:::caution
+If cert-manager isn't installed in the namespace `cert-manager`,
+you have to set `.Values.certManagerNamespace` accordingly, otherwise Epinio installation will fail.
+:::
+
+### Dynamic storage provisionier
+Epinio needs a dynamic storage provisioner with existing **default StorageClass**. You may deploy any storage provisioner
+of your choice, preferably with `ReadWriteMany` (RWX) Access Mode.
+
+:::info
+You may verify that a default StorageClass is available in your cluster in `kubectl get storageclass` output. The default StorageClass is marked by `(default)` string next to its name in the output.
```

Comment:
```suggestion
To verify that your cluster provides a default StorageClass run the command  `kubectl get storageclass`. The default StorageClass is marked with the string `(default)` next to its name in the output list.
```

Same thing as in file `docs/howtos/install_epinio_on_rke.md`

Reply:
resolved by ad889100488915b82ee3c4f30c5c3e6b48298e10